src/main/webapp/app/overview/course-conversations/layout/conversation-messages/conversation-messages.component.ts

```diff
@@ -223,7 +223,9 @@ export class ConversationMessagesComponent implements OnInit, AfterViewInit, OnD
     };
 
     scrollToBottomOfMessages() {
-        this.content.nativeElement.scrollTop = this.content.nativeElement.scrollHeight;
+        setTimeout(() => {
```

Comment:
Can you add a comment here why the timeout is necessary? Otherwise future devs might be confused about the timeout